src/main/java/org/spongepowered/api/world/Chunk.java

```diff
@@ -84,4 +84,39 @@
      * @return Whether or not the chunk unloaded
      */
     boolean unloadChunk();
+
+    /**
+     * Gets the number of ticks players have been present in this chunk, used
+     * for calculation of the regional difficulty factor. In vanilla, it is
+     * increased by the number of players in the chunk every tick, and is capped
+     * at 3,600,000 ticks (50 hours).
+     */
+    int getInhabittedTime();
+
+    /**
+     * 
+     * Gets the regional difficulty factor for this chunk. In vanilla, it is
+     * dependent on the playtime of the world, inhabited time of the chunk, the
+     * phase of the moon, and the current difficulty setting. This number ranges
+     * from 0.75-1.5 on easy, 1.5-4.0 on normal, and 2.25-6.75 on hard.
+     * 
+     * This value is used for display only in vanilla.
+     * 
+     * @return The regional difficulty factor for this chunk
+     */
+    float getRegionalDifficultyFactor();
```

Comment:
Why did you choose floats?
Other implementations might use doubles.

It would be cool to get the min and max values as well.

Why didn't you use a DataManipulator?